server/test/Main.hs

```diff
@@ -2,6 +2,18 @@ module Main (main) where
 
 import Api (app, applyArgs, blake2bHash, estimateTxFees, hashData, hashScript)
 import Cardano.Api qualified as C
+import Cardano.Api.Shelley (
+  ExecutionUnitPrices (
+    ExecutionUnitPrices,
+    priceExecutionMemory,
+    priceExecutionSteps
+  ),
+  ExecutionUnits (ExecutionUnits, executionMemory, executionSteps),
+  Lovelace (Lovelace),
+  ProtocolParameters (..),
+  --  makePraosNonce,
```

Comment:
Please remove

Reply:
It was part of the `extra entropy` issue, fixed.